lib/msal-common/src/config/ClientConfiguration.ts

```diff
@@ -82,6 +82,7 @@ export type AuthOptions = {
  */
 export type SystemOptions = {
     tokenRenewalOffsetSeconds?: number;
+    preventCorsPreflight?: boolean;
```

Comment:
Call this something like `useHeaders` instead?

Reply:
It's a bit misleading, because we still attach headers if this is set to true

Reply:
I am open to changing it but would prefer something more relevant to the config option

Reply:
hmm ok not a big deal. I dont have a better suggestion atm